lighthouse-core/audits/unused-css-rules.js

```diff
@@ -79,6 +89,42 @@ class UnusedCSSRules extends Audit {
   }
 
   /**
+   * Trims stylesheet content down to the first rule-set definition.
+   * @param {string} content
+   * @return {string}
+   */
+  static determineContentPreview(content) {
+    let preview = content
+        .slice(0, PREVIEW_LENGTH * 5)
+        .replace(/( {2,}|\t)+/g, '  ') // remove leading indentation if present
+        .replace(/\n\s+}/g, '\n}') // completely remove indentation of closing braces
+        .trim(); // trim the leading whitespace
+
+    if (preview.length > PREVIEW_LENGTH) {
+      const firstRuleStart = preview.indexOf('{');
+      const firstRuleEnd = preview.indexOf('}');
+
+      if (firstRuleStart === -1 || firstRuleEnd === -1
```

Comment:
Is there anything from https://github.com/ChromeDevTools/devtools-frontend/blob/b84fc38d5618d27b6a5b50d3948cb31a38488bfa/front_end/gonzales/SCSSParser.js we can reuse

not sure, just curious if we can leave some of the parsing/traversal to another project.